at-hoangtran/Week3/Day4/Exercise3-Day4.rb

```diff
@@ -0,0 +1,11 @@
+class HtmlToText
+  def get_html(html)
+    str = html.split('<div>', '</div>', 'br>')
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/3cf28bfecc132fb6fcec164677df66b52ed8042f/at-hoangtran/Week3/Day4/Exercise3-Day4.rb#L3"></span>Lint/UselessAssignment: Useless assignment to variable - <code>str</code>. (https://github.com/rubocop-hq/ruby-style-guide#underscore-unused-vars)</td>
    </tr>
  </tbody>
</table>